mlir/include/mlir/Dialect/AMDGPU/IR/AMDGPU.td

```diff
@@ -656,6 +656,60 @@ def AMDGPU_SwizzleBitModeOp : AMDGPU_Op<"swizzle_bitmode",
   }];
 }
 
+def AMDGPU_PermlanePerm : I32EnumAttr<"PermlanePerm",
```

Comment:
Are we _sure_ we don't want to call this `permlane_swap`? Or are we planning to extend this